odoo/addons/base/tests/test_user_has_group.py

```diff
@@ -249,3 +249,29 @@ def test_implied_groups(self):
 
         with self.assertRaises(ValidationError):
             user_b.write({"groups_id": [(4, group_C.id)]})
+
+    def test_has_group_cleared_cache_on_write(self):
+        Users = self.env["res.users"]
+        self.env.registry._clear_cache()
+        self.assertFalse(Users.pool.cache, "Ensure ormcache is empty")
```

Comment:
The registry is available as `self.registry` on `TransactionCase` instances.
So the variable `Users` is not necessary, and the code can be simplified as
```suggestion
        self.registry._clear_cache()
        self.assertFalse(self.registry.cache, "Ensure ormcache is empty")
```
Similar simplifications can be done below.